libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx

```diff
@@ -49,6 +150,11 @@ const socials: Array<SocialType> = [
     Icon: GithubFill,
     href: 'https://github.com/webb-tools',
   },
+  {
+    name: 'YouTube',
+    Icon: YouTubeFill,
+    href: 'https://github.com/webb-tools',
```

Comment:
I think we can point this to blog videos for now. What do you think @dutterbutter?

https://webb.tools/blog/videos